test/helpers.lua

```diff
@@ -297,12 +297,13 @@ local function dedent(str)
     -- no minimum common indent
     return str
   end
+  local left_indent = (' '):rep(leave_indent or 0)
```

Comment:
I added additional argument to add some indent. The purpose is that I may have six-space indent in the function and yet only three-space indent coming out of `dedent`.